src/utils/win/SkDWrite.cpp

```diff
@@ -29,9 +29,8 @@ static void release_dwrite_factory() {
 
 static void create_dwrite_factory(IDWriteFactory** factory) {
     typedef decltype(DWriteCreateFactory)* DWriteCreateFactoryProc;
-#ifdef RTC_WINDOWS_DESKTOP
```

Comment:
Same here

Reply:
These were changes that we implemented?

Reply:
Yeah, anything with RTC_ is our specific changes needed to get things building for WinUWP. Since UWP has banned APIs, skia might not build without it. That being said, they may have added support for it now but I'm not entirely sure. Do you know if there's native UWP support now?